requirements-test.txt

```diff
@@ -1,7 +1,8 @@
 coveralls
 responses
 freezegun
-openwisp-utils[qa]>=0.2.1
+openwisp-utils[qa]>=0.3.1
+openwisp-users>=0.1.11
```

Comment:
openwisp-users is already in the `requirements.txt`, why have you added it here as well? :smile: